package.json

```diff
@@ -21,17 +21,19 @@
     "axios": "^0.21.1",
     "cross-fetch": "^3.1.4",
     "framer-motion": "^3.2.1",
-    "next": "^10.0.4",
+    "next": "^12.0.7",
+    "npm": "^8.3.0",
```

Comment:
Looks like you've done `npm install npm` 😂

Reply:
No idea...didn't do that for sure 🤣